lib/github_service/commands/remove_label.rb

```diff
@@ -3,7 +3,9 @@ module Commands
     class RemoveLabel < Base
       include IsTeamMember
 
-      UNREMOVABLE = %w[jansa/yes jansa/no].freeze
+      def unremovable_labels
```

Comment:
Maybe make this a class variable so we aren't loading this on every instance?

Reply:
Of course, if needed, you can alias it as an instance method, but I think that it should still be cached on the class level.